README.md

```diff
@@ -216,50 +251,118 @@ Example usage:
   lines so you'll need to be careful to reconstruct this
 - `rails debug:saml['tmp/your_saml_to_debug.txt']`
 
-## Data Loading
+## User roles
+
+There are a few user roles that provide different levels of permissions. The
+abilities of each role are defined in the [ability model](https://github.com/MITLibraries/thing/blob/master/app/models/ability.rb).
+
+`Basic` is the default user role and is assigned when a user self creates an
+account. Self creation is the only supported way to create an account. If we
+have a new staff member, they first need to login to self create an account
+with the basic role and then admin staff can assign them appropriate roles.
+
+`Thesis Processor` is assigned to any users that process theses.
```

Comment:
Should we note that while this role exists in the application, nobody uses it?

Reply:
Oh, good call. I thought Wilder still used it, but I see they're a thesis admin.

Reply:
Do you happen to remember why we still have this role? I can't remember if we just didn't remove it or if we were retaining it for potential future use, i.e. if we have more than one processor and some of them don't need to delete stuff.

Reply:
I think we imagined we needed a distinction in roles between processor and "the processor's manager" kinda thing. It's unclear if our current processor is doing more than expected or our roles were just poorly envisioned. I lean towards thinking we should very likely remove it, but that would be fairly painful (mostly lots of test deletions)

Reply:
Yeah, that's my recollection re: "processor's manager". I do also recall that the role of the processor expanded since we started designing the app.

I'm open to removing it, but it will definitely be painful (test deletions for most controller actions), and more so if we end up actually needing it in the future.